src/views/splash/splash.jsx

```diff
@@ -448,10 +223,15 @@ var Splash = injectIntl(React.createClass({
 var mapStateToProps = function (state) {
     return {
         session: state.session,
-        permissions: state.permissions
+        permissions: state.permissions,
+        rows: state.rows,
+        activity: state.activity,
+        news: state.news,
+        projectCount: state.projectCount,
+        homepageCache: state.homepageCache
```

Comment:
Something to think about — do more work in `mapStateToProps` so that we only pass in the data we need rather than broad categories. E.g., instead of just `session`, pass in the user if it exists. If you need to know if the session is loaded, then set another prop, `sessionLoaded` for that purpose.  That will help get rid of the weird doubling `this.props.stateKey.stateKey` everywhere, and can help us be more explicit about what we're using, so we'll know better when something is unused and can get removed from the state.